ethportal-peertest/src/scenarios/eth_rpc.rs

```diff
@@ -7,6 +8,7 @@ use crate::{
 use trin_core::jsonrpc::types::Params;
 
 pub fn test_eth_get_block_by_hash(_peertest_config: PeertestConfig, peertest: &Peertest) {
+    info!("Testing eth_getBlockByHash");
```

Comment:
I used it for debugging, but left it in purposefully since I think it's useful. All of our test runs in `jsonrpc.rs` have `info` logs stating which test is currently being run. Keeping this `info` log here helps communicate which test is currently being run / failing